engine/components/sql/src/main/scala/pl/touk/nussknacker/sql/db/schema/DbMetaDataProvider.scala

```diff
@@ -23,10 +25,28 @@ class DbMetaDataProvider(getConnection: () => Connection) {
       } finally statement.close()
     } finally conn.close()
   }
+
+  def getSchemaDefinition(): SchemaDefinition = {
+    val connection = getConnection()
+    val tables = connection.getMetaData.getTables(null, "PUBLIC", "%", Array("TABLE").map(_.toString))
+    val results = new util.ArrayList[String]()
+    val columnNameIndex = 3
+    while (tables.next()) {
```

Comment:
maybe some scala conversions would be easier?

Reply:
Done